war/src/main/less/modules/panes-and-bigtable.less

```diff
@@ -0,0 +1,143 @@
+/* pane */
+
+.pane-frame {
+  border: var(--pane-border-width) solid var(--pane-border-color);
+  border-radius: 4px;
+}
+
+.pane-header, .pane-footer {
+  padding: 8px 0px;
+  color: var(--pane-header-color);
+  background: var(--pane-header-bg);
+  border: var(--pane-border-width) solid var(--pane-header-border-color);
+  border-left: none;
+  border-right: none;
+  color: var(--pane-text-color);
+}
+
+.pane-header {
+  border-top: none;
+  border-top-left-radius: 4px;
+  border-top-right-radius: 4px;
+}
+
+.pane-footer {
+  border-bottom: none;
+  border-bottom-left-radius: 4px;
+  border-bottom-right-radius: 4px;
+}
+
+.pane td {
+  padding: 0.25rem;
+  vertical-align: middle;
+}
+
+table.pane {
+  width: 100%;
+  border-collapse: collapse;
+  border: var(--pane-border-width) solid var(--pane-border-color);
+}
+
+td.pane {
+  padding: 0.25rem;
+  vertical-align: middle;
+}
+
+table.stripped tr:nth-of-type(even) {
+  background: var(--even-row-color);
+}
+
+table.stripped-even tr:nth-child(even) {
+  background: var(--even-row-color);
+}
+table.stripped-odd tr:nth-child(odd) {
+  background: var(--even-row-color);
+}
+table.stripped tr:hover, table.stripped-even tr:hover, table.stripped-odd tr:hover {
+  background: var(--table-striped-bg--hover) !important;
+}
+
+div.pane-header {
+  font-weight: var(--pane-header-font-weight);
+  padding-right: 24px;
+}
+
+div.pane-header .collapse {
+  float: right;
+  margin-left: 3px;
+}
+
+th.pane {
+  font-weight: var(--pane-header-font-weight);
+}
+
+/* Bigtable */
+
+table.bigtable {
+  border: none; // Needs to be set AFTER the table.pane declaration
+}
+
+/* For non-full screen table */
+
+.bigtable.width-auto {
+  width: auto;
+}
+
+// Keep this for tables still wrapped by .pane-frame
+.pane-frame table, .pane-frame .bigtable tr {
+  border: none; /* Border will be provided by the pane-frame */
```

Comment:
This rule appears to have caused https://issues.jenkins.io/browse/JENKINS-63677 which makes all progress bars in Jenkins completely useless.